### DISCUSSION TITLE
How to install?

### DISCUSSION THREAD

**Discussion by Rumik** [trakt-tools/universal-trakt-scrobbler#108] [Q&A]
Hi there,

how do we install this, now that it's not on the Chrome web store?

**Comment by Trigger911** [comment]
https://github.com/trakt-tools/universal-trakt-scrobbler/releases/tag/beta ... follow the instructions on the primary page documentation I am sure they probably don't want to help if you don't research. Sorry, that came off harsh but you need to build it yourself.
